pkg/web/apikeys.go

```diff
@@ -53,7 +53,6 @@ func (s *Server) ListAPIKeys(c *gin.Context) {
 		Offered:  []string{binding.MIMEJSON, binding.MIMEHTML},
 		Data:     out,
 		HTMLName: "apikey_list.html",
-		HTMLData: scene.New(c).WithAPIData(out),
```

Comment:
While working on this I realized that we may not need this in the list partials anymore since we're planning to remove the actions menu from the tables.

Reply:
That's fine, though I don't see any harm on having the scene context as html data standard; in case we change things in the future.